keyboards/kprepublic/bm60hsrgb_poker/rev2/keymaps/TheChilledDejw/rules.mk

```diff
@@ -0,0 +1,33 @@
+# MCU name
```

Comment:
Most of this should be removed.  Only the settings that need to be changed should be included here.